### ISSUE TITLE
run-tests pipeline executes prior to the re-deployment of the ecosystem

### ISSUE THREAD

**Issue by techcobweb** [galasa-dev/projectmanagement#1842]
# Bug

The run-tests pipeline runs before the ecosystem has been re-deployed.

It was supposed to be scheduled after the ecosystem re-deploy, but appears to run at the incorrect time in the chain ?

Only noticed this recently because the ecosystem was down, so these tests failed, and shortly after the ecosystem was re-deployed, and the tests worked after that on a re-run.

# Tasks

-  [x] Rearrange pipeline chain so that the ecosystem is redeployed before run-tests is called

**Comment by techcobweb** [comment]
cc: @jadecarino

**Comment by jadecarino** [comment]
@techcobweb Hi Mike, thanks for pointing this out. The run-tests pipeline was previously (in the old builds with Jenkins) placed before redeploying the ecosystem, as it was meant to catch any issues before that code was deployed to prod. We kept this same order in our new build system with Tekton. However, the benefit of that has now disappeared (we used to test a staging branch and that would then be promoted to prod, we no longer do this) and now the order is actually getting us into these situations where the tests are failing and so the fixes are unable to make it to the ecosystem.
